personality/se_dpm/puppet/info_user_proxy.pan

```diff
@@ -22,21 +22,36 @@ include 'components/cron/config';
 variable DPM_HOSTPROXY_CRON = SEDPM_INFO_USER + "-hostproxy";
 variable DPM_HOSTPROXY_TMP = GLITE_LOCATION_VAR + "/dpm/hostproxy.$$";
 variable DPM_HOSTPROXY_FILE = GLITE_LOCATION_VAR + "/dpm/hostproxy." + SEDPM_INFO_USER;
-variable DPM_HOSTPROXY_CMD = "mkdir -p /var/lib/ldap/.globus && /bin/cp /etc/grid-security/hostcert.pem /var/lib/ldap/.globus/usercert.pem && /bin/cp /etc/grid-security/hostkey.pem /var/lib/ldap/.globus/userkey.pem && chown -R " + SEDPM_INFO_USER + " /var/lib/ldap/.globus";
-"/software/components/cron/entries" = {
-    push(dict(
+variable DPM_HOSTPROXY_CMD = {
+    command = "mkdir -p /var/lib/ldap/.globus && ";
+    command = command + "/bin/cp /etc/grid-security/hostcert.pem /var/lib/ldap/.globus/usercert.pem && ";
+    command = command + "/bin/cp /etc/grid-security/hostkey.pem /var/lib/ldap/.globus/userkey.pem && ";
+    command = command + "chown -R " + SEDPM_INFO_USER + " /var/lib/ldap/.globus";
+    command;
+};
```

Comment:
This would be a slight improvement:

```pan
variable DPM_HOSTPROXY_CMD = {
    "mkdir -p /var/lib/ldap/.globus && " +
    "/bin/cp /etc/grid-security/hostcert.pem /var/lib/ldap/.globus/usercert.pem && " +
    "/bin/cp /etc/grid-security/hostkey.pem /var/lib/ldap/.globus/userkey.pem && " +
    "chown -R " + SEDPM_INFO_USER + " /var/lib/ldap/.globus";
};
```

This would be even better, but requires panc 10.4 or later:
```pan
variable DPM_HOSTPROXY_CMD = join(" && ",
    "mkdir -p /var/lib/ldap/.globus",
    "/bin/cp /etc/grid-security/hostcert.pem /var/lib/ldap/.globus/usercert.pem",
    "/bin/cp /etc/grid-security/hostkey.pem /var/lib/ldap/.globus/userkey.pem",
    format("chown -R %s /var/lib/ldap/.globus", SEDPM_INFO_USER),
);
```